lib/pbench/agent/tool_data_sink.py

```diff
@@ -221,7 +221,7 @@ def launch(self):
                 )
                 return 0
 
-            args = ["podman", "pull", "prom/prometheus"]
+            args = ["podman", "pull", "quay.io/prometheus/prometheus"]
```

Comment:
In scrum I think Pete had suggested that we consider moving this to a config option, which would allow easily substituting another build of Prometheus tools if necessary for some special purpose. That might be worth considering; but I don't think it needs to gate the "quick fix".

Reply:
Tracked in issue #2016.